packages/resource-deployment/scripts/batch-account-create.sh

```diff
@@ -74,6 +78,10 @@ echo "Setting up batch account $batchAccountName"
 # Configure Azure subscription account to support Batch account in user subscription mode
 . "${0%/*}/account-set-batch-app.sh"
 
+setParameterFilePath
+
+. "${0%/*}/delete-outdated-pools.sh"
```

Comment:
you should be running this command after disabling the schedule & waiting for noodes to be idle.

you could update the recreate pools script to do this. -https://github.com/microsoft/accessibility-insights-service/blob/master/packages/resource-deployment/scripts/recreate-vmss-for-pools.sh

checkIfVmsssAreOld could also look at the pool config & delete all pools if one of them is outdated.
And move the recreate pools script to be run before batch-account-create.sh.

Reply:
you can remove this deployment script change & work on it in a separate pr. Lets merge the changes for the pool config now.